src/util/fields.py

```diff
@@ -676,7 +676,7 @@ def convert_field_to_html(cr, model, field, skip_inherit=()):
 
     # Update ir.default
     if table_exists(cr, "ir_default"):
-        json_value_html = pg_text2html("json_value")
+        json_value_html = pg_text2html("json_value::json #>> '{}'").replace("<p>", '"<p>').replace("</p>", '</p>"')
```

Comment:
The point is to avoid this replace.
```suggestion
        json_value_html = pg_text2html("json_value::json #>> '{}'")
```